platform_versioned_docs/version-24.3/studios/index.mdx

```diff
@@ -87,47 +87,47 @@ The default user is the `root` account. The following [conda-forge](https://cond
 
 To see the list of all Xpra image templates available, see [public.cr.seqera.io/repo/platform/data-studio-xpra][ds-xpra].
 
-## Session statuses
+## Studio session statuses
 
-Data studios have the following possible statuses:
+Studios have the following possible statuses:
 
-- **building**: When a custom environment is building the template image for a new data studio session. The [Wave] service performs the build action. For more information on this status, see [Inspect custom container template build status][build-status].
+- **building**: When a custom environment is building the template image for a new Studio session. The [Wave] service performs the build action. For more information on this status, see [Inspect custom container template build status][build-status].
 - **build-failed**:  When a custom environment build has failed. This is a non-recoverable error. Logs are provided to assist with troubleshooting. For more information on this status, see [Inspect custom container template build status][build-status].
-- **starting**: The data studio is initializing.
-- **running**: When a data studio session is **running**, you can connect to it, copy the data studio URL, or stop it. In addition, the session can continue to process requests/run computations in the absence of an ongoing connection.
+- **starting**: The Studio is initializing.
+- **running**: When a Studio session is **running**, you can connect to it, copy the URL, or stop it. In addition, the session can continue to process requests/run computations in the absence of an ongoing connection.
 - **stopping**: The recently-running session is in the process of being stopped.
-- **stopped**: When a session is stopped, the associated compute resources are deallocated. You can start or delete the data studio when it's in this state.
-- **errored**: This state most often indicates that there has been an error starting the data studio session but it is in a **stopped** state. There might be errors reported by the session itself but these will be overwritten with a **running** status if the data studio session is still running.
+- **stopped**: When a session is stopped, the associated compute resources are deallocated. You can start or delete the Studio when it's in this state.
+- **errored**: This state most often indicates that there has been an error starting the Studio session but it is in a **stopped** state. There might be errors reported by the session itself but these will be overwritten with a **running** status if the session is still running.
 
   :::tip
   If you encounter an error with the public preview release of Studios, [contact Seqera support][contact].
   :::
 
-## Session data links
+## Studio session data links
 
 You can configure a Studio to mount one or more data links, where cloud buckets that you have configured in your compute environment are read-only, or read-write available to the session.
```

Comment:
```suggestion
You can configure a Studio session to mount one or more data links, where cloud buckets that you have configured in your compute environment are read-only, or read-write available to the session.
```

Reply:
@justinegeffen I thought we agreed it was either "Studios" (proper noun) or "studio". In this case, shouldn't it be `studio session` and not `Studio session`? I guess it could be `Studios session`, but that doesn't feel right.